### DISCUSSION TITLE
Audio preview featire...

### DISCUSSION THREAD

**Discussion by emigrating** [advplyr/audiobookshelf#2330] [Ideas]
Often, when I'm spending my monthly Audible credits (or simply browsing the "sales") I press the preview button before anything else to decide whether I can stand the narrator or not. If I find the voice bad I quickly skip to the next one, but if I find it okay I'll let it play while checking out the blurb and a couple reviews to really make up my mind about it.

Something similar would be great in ABS. Especially in larger libraries shared [and sourced] with friends and family where titles can often be very diverse.

**Comment by nichwall** [comment]
Progress is not synced for the first... 60 seconds? I forgot exactly how long.
This is so you can preview without the progress being stored/showing up on your Continue Listening shelf.

I'm not sure if seeking within the book automatically saves (skipping the "no saving" time)

**Reply by hobesman** [reply]
Is this universal? I just played a new book for about 10 seconds in the mobile app. Paused it. And closed the player. Now that book is in my continue listening shelf.

**Reply by advplyr** [reply]
It's currently 15 seconds on the mobile apps with no additional delay on first open. For android on a metered network connection it will be 60 seconds sync and on iOS for devices in low power mode it will be 60 seconds also.

**Reply by hobesman** [reply]
Is that measured in book time or real time? E.g. listening at faster speed does or does not change those? 

And does it start from pressing play, or from when playback starts (which could be a significant difference on a slow connection...)?

**Reply by advplyr** [reply]
Real time. The timer should start when the audio begins playing. For android we are listening to the "isPlayingChanged" event listener https://exoplayer.dev/listening-to-player-events.html#playback-state-changes
When the audio is buffering it is not considered as playing

**Reply by hobesman** [reply]
Interesting. Well, I still agree with OP that a dedicated preview option is worth considering.

**Reply by advplyr** [reply]
There is a feature request open for it already https://github.com/advplyr/audiobookshelf/issues/1032

**Reply by emigrating** [reply]
> It's currently 15 seconds on the mobile apps with no additional delay on first open. For android on a metered network connection it will be 60 seconds sync and on iOS for devices in low power mode it will be 60 seconds also.

I dont think this is working for me at all. At least not in the web app. If I as much as hit play on a title it seems to be recorded as a session and track location. I always seem to have to clean up my history,

I upvoted the existing feature request (which I didn't find when I searched before posting but...)..

I'd say just add a "preview" button under the thumbnail or something and have that play from minute 3 to 6 or something. With no history recorded.

**Reply by advplyr** [reply]
@emigrating What version of the server are you using?

**Reply by emigrating** [reply]
2.5.0
